CHANGELOG.md

```diff
@@ -6,8 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v16.0.0] 2023-05-09
+
 ### MIGRATION notes
 
+#### PI release version
+
+When migrating directly to v16 from prior releasesolder that V15, please make sure to
+read through all prior release migrations.
```

Comment:
```suggestion
read through all prior release notes.
```
Just being careful about the word "migration" unless that's actually what we mean here.